### PR TITLE
Prefix dashboard IDs with package names

### LINKED ISSUE #319: Prefix dashboard IDs with package names
We need to ensure that all packages' Kibana dashboards have IDs that start with package names.

This is where things stand at the moment:

```
Checking package apache...
Dashboards: 
  [PASS] apache-Metrics-Apache-HTTPD-server-status-ecs.json
  [PASS] apache-Logs-Apache-Dashboard-ecs.json

Checking package aws...
Dashboards: 
  [PASS] aws-234aeda0-43b7-11e9-8697-530f39afc6eb.json
  [PASS] aws-e74bf320-b3ce-11e9-87a4-078dbbae220d.json
  [PASS] aws-c2b1cbc0-6891-11ea-b0ac-95d4ecb1fecd.json
  [PASS] aws-68ba7bd0-20b6-11ea-8f72-2f8d21e50b0c.json
  [PASS] aws-fac28650-7349-11e9-816b-07687310a99a.json
  [PASS] aws-a096b830-4762-11e9-8062-c98a86cb6f94.json
  [PASS] aws-0eb5a6a0-694f-11ea-b0ac-95d4ecb1fecd.json
  [PASS] aws-67c9f900-693e-11ea-b0ac-95d4ecb1fecd.json
  [PASS] aws-3367c170-921f-11e9-aa19-159bf182e06f.json
  [PASS] aws-c5846400-f7fb-11e8-af03-c999c9dea608-ecs.json
  [PASS] aws-44ce4680-b7ba-11e9-8349-f15f850c5cd0.json
  [PASS] aws-15503340-4488-11ea-ad63-791a5dc86f10.json
  [PASS] aws-4746e000-bacd-11e9-9f70-1f7bda85a5eb.json
  [PASS] aws-d17b1000-17a4-11ea-8e91-03c7047cbb9d.json
  [PASS] aws-9c09cd20-7399-11ea-a345-f985c61fe654.json
  [PASS] aws-e6776b10-1534-11ea-841c-01bf20a6c8ba.json
  [PASS] aws-3af47420-3e7b-11ea-bb0a-69c3ca1d410f.json
  [PASS] aws-7ac8e1d0-28d2-11ea-ba6c-49a884eb104f.json
  [PASS] aws-917a07b0-178e-11ea-8650-fb606deb5be4.json

Checking package checkpoint...
Dashboards: 

Checking package cisco...
Dashboards: 
  [PASS] cisco-a555b160-4987-11e9-b8ce-ed898b5ef295.json

Checking package crowdstrike...
Dashboards: 

Checking package fortinet...
Dashboards: 

Checking package haproxy...
Dashboards: 
  [FAIL] 4b555c30-47dd-11e8-bc13-1397384faad3-ecs.json
  [FAIL] 8cc50a50-47e0-11e8-bc13-1397384faad3-ecs.json
  [FAIL] d5878d00-47c5-11e8-bc13-1397384faad3-ecs.json
  [FAIL] e9057ae0-47c5-11e8-bc13-1397384faad3-ecs.json
  [FAIL] 3560d580-aa34-11e8-9c06-877f0445e3e0-ecs.json
  [FAIL] 0836a4b0-47bd-11e8-bc13-1397384faad3-ecs.json
  [FAIL] 9151c900-471d-11e8-bc13-1397384faad3-ecs.json

Checking package iis...
Dashboards: 
  [PASS] iis-4b975820-85a1-11ea-91bc-ab084c7ec0e7.json
  [PASS] iis-2c171500-858b-11ea-91bc-ab084c7ec0e7.json
  [PASS] iis-b4108810-861c-11ea-91bc-ab084c7ec0e7.json
  [PASS] iis-4278ad30-fe16-11e7-a3b0-d13028918f9f-ecs.json
  [PASS] iis-ebc23240-8572-11ea-91bc-ab084c7ec0e7.json

Checking package juniper...
Dashboards: 

Checking package kafka...
Dashboards: 
  [PASS] kafka-ea488d90-8e63-11e8-8fa2-3d5f811fbd0f-ecs.json
  [PASS] kafka-943caca0-87ee-11e7-ad9c-db80de0bf8d3-ecs.json

Checking package kubernetes...
Dashboards: 
  [PASS] kubernetes-5e649d60-9901-11e9-ba57-b7ab4e2d4b58.json
  [PASS] kubernetes-f5ab5510-9c94-11e9-94fd-c91206cd5249.json
  [PASS] kubernetes-AV4RGUqo5NkDleZmzKuZ-ecs.json
  [PASS] kubernetes-97312060-9c1b-11e9-9dc8-fd27291d427f.json
  [PASS] kubernetes-af7225b0-5794-11e8-afa2-e9067ea62228-ecs.json

Checking package linux...
Dashboards: 
  [FAIL] system-c431f410-f9ac-11e9-90e8-1fb18e796788.json

Checking package log...
Dashboards: 

Checking package microsoft...
Dashboards: 

Checking package mongodb...
Dashboards: 
  [PASS] mongodb-abcf35b0-0a82-11e8-bffe-ff7d4f68cf94-ecs.json
  [PASS] mongodb-Metrics-MongoDB-ecs.json

Checking package mysql...
Dashboards: 
  [PASS] mysql-57b3fb50-6309-11ea-a83e-25b8612d00cc.json
  [PASS] mysql-Logs-MySQL-Dashboard-ecs.json

Checking package netflow...
Dashboards: 
  [PASS] netflow-acd7a630-0c71-4840-bc9e-4a3801374a32.json
  [PASS] netflow-94972700-de4a-4272-9143-2fa8d4981365.json
  [PASS] netflow-34e26884-161a-4448-9556-43b5bf2f62a2.json
  [PASS] netflow-feebb4e6-b13e-4e4e-b9fc-d3a178276425.json
  [PASS] netflow-c64665f9-d222-421e-90b0-c7310d944b8a.json
  [PASS] netflow-38012abe-c611-4124-8497-381fcd85acc8.json
  [PASS] netflow-14387a13-53bc-43a4-b9cd-63977aa8d87c.json
  [PASS] netflow-77326664-23be-4bf1-a126-6d7e60cfc024.json

Checking package nginx...
Dashboards: 
  [PASS] nginx-023d2930-f1a5-11e7-a9ef-93c69af7b129-ecs.json
  [PASS] nginx-55a9e6e0-a29e-11e7-928f-5dbe6f6f5519-ecs.json
  [PASS] nginx-046212a0-a2a1-11e7-928f-5dbe6f6f5519-ecs.json

Checking package o365...
Dashboards: 
  [FAIL] 712e2c00-685d-11ea-8d6a-292ef5d68366.json

Checking package okta...
Dashboards: 
  [FAIL] 749203a0-67b1-11ea-a76f-bf44814e437d.json

Checking package panw...
Dashboards: 
  [FAIL] e40ba240-7572-11e9-976e-65a8f47cc4c1.json
  [FAIL] 772964e0-7591-11e9-aacf-79a3704914a0.json

Checking package postgresql...
Dashboards: 
  [PASS] postgresql-158be870-87f4-11e7-ad9c-db80de0bf8d3-ecs.json
  [PASS] postgresql-e4c5f230-87f3-11e7-ad9c-db80de0bf8d3-ecs.json
  [PASS] postgresql-4288b790-b79f-11e9-a579-f5c0a5d81340.json

Checking package prometheus...
Dashboards: 
  [PASS] prometheus-6a9b80c0-b2ed-11e9-9a23-67ee28886a4b.json

Checking package rabbitmq...
Dashboards: 
  [PASS] rabbitmq-AV4YobKIge1VCbKU_qVo-ecs.json

Checking package redis...
Dashboards: 
  [PASS] redis-7fea2930-478e-11e7-b1f0-cb29bac6bf8b-ecs.json
  [PASS] redis-28969190-0511-11e9-9c60-d582a238e2c5-ecs.json
  [PASS] redis-AV4YjZ5pux-M-tCAunxK-ecs.json

Checking package suricata...
Dashboards: 
  [FAIL] 05268ee0-86d1-11e8-b59d-21efb914e65c-ecs.json
  [FAIL] 78289c40-86da-11e8-b59d-21efb914e65c-ecs.json

Checking package system...
Dashboards: 
  [PASS] system-0d3f2380-fa78-11e6-ae9b-81e5311e8cab-ecs.json
  [PASS] system-277876d0-fa2c-11e6-bbd3-29c986c96e5a-ecs.json
  [PASS] system-CPU-slash-Memory-per-container-ecs.json
  [PASS] system-79ffd6e0-faa0-11e6-947f-177f697178b8-ecs.json
  [PASS] system-5517a150-f9ce-11e6-8115-a7c18106d86a-ecs.json
  [PASS] system-Metricbeat-system-overview-ecs.json
  [PASS] system-Filebeat-syslog-dashboard-ecs.json

Checking package windows...
Dashboards: 
  [FAIL] 71f720f0-ff18-11e9-8405-516218e3d268.json
  [FAIL] 01c54730-fee6-11e9-8405-516218e3d268.json
  [FAIL] d401ef40-a7d5-11e9-a422-d144027429da.json
  [FAIL] 8223bed0-b9e9-11e9-b6a2-c9b4015c4baf.json
  [FAIL] d9eba730-c991-11e7-9835-2f31fe08873b-ecs.json
  [FAIL] 035846a0-a249-11e9-a422-d144027429da.json
  [FAIL] bb858830-f412-11e9-8405-516218e3d268.json
  [FAIL] Winlogbeat-Dashboard-ecs.json
  [FAIL] c77e06c0-9e7c-11ea-af6f-cfdb1ee1d6c8.json
  [FAIL] bae11b00-9bfc-11ea-87e4-49f31ec44891.json
  [PASS] windows-d9eba730-c991-11e7-9835-2f31fe08873b-ecs.json
  [FAIL] f49f3170-9ffc-11ea-87e4-49f31ec44891.json

Checking package zeek...
Dashboards: 
  [FAIL] 7cbb5410-3700-11e9-aa6d-ff445a78330c.json

Checking package zookeeper...
Dashboards: 
  [PASS] zookeeper-467207a0-231e-11e9-bb66-8baac426dfd4-ecs.json

Checking package zoom...
Dashboards: 
```

### PR DESCRIPTION
**Pull request by ycombinator**
## What does this PR do?

This PR iterates over all integrations packages and checks if any of their dashboard assets have IDs that do **not** start with the package's name. If such assets are found, their IDs are fixed to start with the package name and any references to those IDs from other asset files are updated.

Script used to make this change: https://gist.github.com/ycombinator/ff4cd5cd4c1fca6f1ae82a496c02e190

## Related issues

- Resolves #319

### CODE REVIEW

File: packages/linux/kibana/dashboard/linux-system-c431f410-f9ac-11e9-90e8-1fb18e796788.json [hunk 1]
```diff
@@ -133,7 +133,7 @@
         "title": "[Metrics Linux] Host Services Overview",
         "version": 1
     },
-    "id": "system-c431f410-f9ac-11e9-90e8-1fb18e796788",
```
Comment (ycombinator) [inline packages/linux/kibana/dashboard/linux-system-c431f410-f9ac-11e9-90e8-1fb18e796788.json:136]: @fearful-symmetry does this change look right to you?

cc: @urso @andresrc

**Review by mtojek — APPROVED** [review]
CI build is green. LGTM!

File: packages/windows/kibana/dashboard/windows-d9eba730-c991-11e7-9835-2f31fe08873b-ecs.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
                 "highlightAll": true,
                 "query": {
                     "language": "kuery",
-                    "query": ""
+                    "query": "(data_stream.dataset:windows.application OR data_stream.dataset:windows.forwarded OR data_stream.dataset:windows.powershell OR data_stream.dataset:windows.powershell_operational OR data_stream.dataset:windows.security OR data_stream.dataset:windows.sysmon_operational OR data_stream.dataset:windows.system)"
```
Comment (ycombinator) [inline packages/windows/kibana/dashboard/windows-d9eba730-c991-11e7-9835-2f31fe08873b-ecs.json:11]: Hey @leehinman, @narph,  this PR has been merged but I just wanted to double check that this change to the `windows` package is okay.

File: packages/windows/kibana/dashboard/windows-d9eba730-c991-11e7-9835-2f31fe08873b-ecs.json [hunk 3]
```diff
@@ -121,6 +121,9 @@
     "migrationVersion": {
         "dashboard": "7.3.0"
     },
+    "namespaces": [
+        "default"
+    ],
```
Comment (ycombinator) [inline packages/windows/kibana/dashboard/windows-d9eba730-c991-11e7-9835-2f31fe08873b-ecs.json:126]: Hey @leehinman, @narph,  this PR has been merged but I just wanted to double check that this change to the `windows` package is okay.

File: packages/windows/kibana/dashboard/windows-d9eba730-c991-11e7-9835-2f31fe08873b-ecs.json (hunk 2 above)
Comment (leehinman) [reply packages/windows/kibana/dashboard/windows-d9eba730-c991-11e7-9835-2f31fe08873b-ecs.json:11]: I think we can drop windows.application & windows.system, those will end up in the system package.  This probably means I need to go through and fix up the others as well.

File: packages/windows/kibana/dashboard/windows-d9eba730-c991-11e7-9835-2f31fe08873b-ecs.json (hunk 3 above)
Comment (leehinman) [reply packages/windows/kibana/dashboard/windows-d9eba730-c991-11e7-9835-2f31fe08873b-ecs.json:126]: LGTM

File: packages/windows/kibana/dashboard/windows-d9eba730-c991-11e7-9835-2f31fe08873b-ecs.json (hunk 2 above)
Comment (ycombinator) [reply packages/windows/kibana/dashboard/windows-d9eba730-c991-11e7-9835-2f31fe08873b-ecs.json:11]: Thanks for checking, @leehinman. Would you mind putting up a PR with the relevant changes (and a package version bump)? That'll let me move forward with https://github.com/elastic/package-storage/pull/495 for now, knowing that we'll end up with another, similar PR after your changes go in.

Comment (ycombinator) [reply packages/windows/kibana/dashboard/windows-d9eba730-c991-11e7-9835-2f31fe08873b-ecs.json:11]: I guess a better question to ask is: given that this PR is merged and a new version of this package is about to be published to the `snapshot` package registry (https://github.com/elastic/package-storage/pull/495), is it safe to go ahead with that publish PR that contains the above query?

Comment (leehinman) [reply packages/windows/kibana/dashboard/windows-d9eba730-c991-11e7-9835-2f31fe08873b-ecs.json:11]: Yep, safe to go ahead.

**Pull request merged**